client/changes/VERSION_COMPAT

```diff
@@ -9,3 +9,5 @@
 # BEGIN DEPENDENCY LIST -------------------------
 # leap.foo.bar>=x.y.z
 pysqlcipher>2.6.3
+leap.common>=4.0
```

Comment:
I think that we should bump the `leap.common` version after the events api change.